packages/commerce-sdk-react/src/hooks/ShopperBaskets/paramKeys.ts

```diff
@@ -1,27 +0,0 @@
-/*
- * Copyright (c) 2024, Salesforce, Inc.
- * All rights reserved.
- * SPDX-License-Identifier: BSD-3-Clause
- * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
- */
-
-const getBasket = ['organizationId', 'basketId', 'siteId', 'locale'] as const
-
-const getPaymentMethodsForBasket = ['organizationId', 'basketId', 'siteId', 'locale'] as const
-const getPriceBooksForBasket = ['organizationId', 'basketId', 'siteId'] as const
-const getShippingMethodsForShipment = [
-    'organizationId',
-    'basketId',
-    'shipmentId',
-    'siteId',
-    'locale'
-] as const
-const getTaxesFromBasket = ['organizationId', 'basketId', 'siteId'] as const
-
-export default {
-    getBasket,
-    getPaymentMethodsForBasket,
```

Comment:
@kevinxh  The ticket asks us to consider whether the change will be a breaking one or not? What specifically made us to think that?

So far I have not seen anything concerning. For example, this file deletion should be ok, right? These param keys were not publicly exported.

Reply:
I think so.

We specifically exports the public interface from the [index file](https://github.com/SalesforceCommerceCloud/pwa-kit/blob/develop/packages/commerce-sdk-react/src/index.ts). Anything not exported from that file is considered private.